### PR TITLE
Add ability to select a single space

### PR DESCRIPTION
**Pull request by Kirisummer**
While using the script I needed to export a single space, yet it wasn't possible (or I've missed something).
Patch adds `--space` argument that may be used to match spaces by their keys

### CODE REVIEW

File: confluence-markdown-export.py [hunk 1]
```diff
@@ -16,7 +16,8 @@ class ExportException(Exception):
 
 
 class Exporter:
-    def __init__(self, url, username, token, out_dir, no_attach):
+    def __init__(self, url, username, token, out_dir, space, no_attach):
+        print(space)
```
Comment (gergelykalman) [inline confluence-markdown-export.py:20]: Please remove this, otherwise it looks good

**Review by gergelykalman — CHANGES_REQUESTED** [review]
Please remove the print statement, and this is good to go

**Comment by gergelykalman** [comment]
I ended doing this in a separate branch, thank you! :)

**Comment by Kirisummer** [comment]
Sorry, I missed the notification :)

**Pull request merged**